src/index.js

```diff
@@ -2,15 +2,14 @@
 
 const path = require('path')
 
-const loaderUtils = require('loader-utils')
-
-const parseOptions = require('./options')
-const validateOptions = require('schema-utils')
+const { getOptions } = require('@webpack-utilities/loader')
```

Comment:
Don't use this package, we don't have time support this package, also he use own logic for parse query, in webpack@5 logic will be change in some places (loader-utils will be update too)

Reply:
> Is there any discussion/documentation about the (planned) changes to the Loader (Runner) API ? (Guessing no)

Only in private messages

In fact, it is only now that work has begun on who is responsible for what, although this should have been done much earlier. And frankly it's bad that we lost you as contributor, I would be glad to see if you would return back. 

> I'm simply sick and tired of working on xyz and at the end of the day it is for nothing concrete because of organisational issues/no feedback and/but on the other hand to top it still every time someone bluntly chimes in and does his/her own thing anyways causing nothing but havoc and burden.

Right now I perform this function and opened for any communication and feedback. 

I know that we have a lot of legacy stuff in many loaders/utils/plugins, which should be removed/refactored, and feel free to do this. Also `webpack-default` is just a repository that helps make the code more consistently. We do not necessarily have to release a huge update (`webpack-default` + breaking change) in one major release - the most important thing is stable code.